index.html

```diff
@@ -103,13 +103,12 @@ <h1>Welcome to DJURPARK</h1>
                 </div>
 
                 <div class="animal-container" id="animal-container">
+                    <img src="./images/Echidna.jpg" class="animal-image"> 
                     <h1 class="animal-name"></h1>
```

Comment:
Can you change this to h2 instead of h1, since h1 is going to be used for the page title.